purchase_order_product_recommendation/wizards/purchase_order_recommendation.py

```diff
@@ -376,5 +376,6 @@ def _prepare_new_po_line(self, sequence):
         return {
             'order_id': self.wizard_id.order_id.id,
             'product_id': self.product_id.id,
+            'partner_id': self.partner_id.id,
```

Comment:
Always add a comment for conflicting things:
```suggestion
            # set this related field manually, as there's a glitch
            # in ORM that doesn't set its initial value
            'partner_id': self.partner_id.id,
```